perfkitbenchmarker/linux_packages/ycsb.py

```diff
@@ -271,8 +274,8 @@ def IsHeadOfResults(line):
   if result_string.startswith('YCSB Client 0.'):
     client_string = result_string
     command_line = next(fp).strip()
-    if not command_line.startswith('Command line:'):
-      raise IOError('Unexpected second line: {0}'.format(command_line))
+#    if not command_line.startswith('Command line:'):
+#      raise IOError('Unexpected second line: {0}'.format(command_line))
```

Comment:
You're probably removing this due to the version change. Is there any other check that should be here?

Reply:
I ran some tests with different ycsb versions to check what is dumped out. 
Up to version 0.7.0., we get a result string starting with 'YCSB Client 0.7.0', followed by the 'Command line: '... .
After version 0.7.0. up till version 0.12.0, we get a result string starting with '[OVERALL]' and the data starts right away.
At version 0.13.0, we get 'YCSB Client 0.13.0', followed by 'Loading workload...' BEFORE the tests get run, which means that IsHeadOfResult function is broken with version 13.
Version 13 is in pre-release the the latest version is actually version 12. I would recommend you use version 12 as 13 might get more changes still. If you insist on using version 13, I'd say you fix IsHeadOfResult instead of commenting out the two lines above.

Reply:
this was for a workaround when it would just cause command_line error for anything past 0.9.0.  i changed it back to original.